core/src/main/kotlin/io/github/serpro69/kfaker/provider/misc/ParameterInfo.kt

```diff
@@ -0,0 +1,25 @@
+package io.github.serpro69.kfaker.provider.misc
+
+import kotlin.reflect.KParameter
+
+/**
+ * Provides additional informations about Class parameter to custom defined generators.
+ * The reason why is not used KParameter is that you will want to provide
```

Comment:
I like having this as separate class instead of using `KParameter` for the opposite reason - to not expose everything that `KParameter` gives if it's not needed :D

Reply:
😘